app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -405,12 +419,22 @@ def handle_unsupported_secondary_id(enrollment, response)
     )
   end
 
+  # MW: This could be inlined but it feels a little cleaner this way:
+  def fraud_pending?(enrollment)
+    enrollment.profile&.fraud_pending_reason.present?
+  end
+
   def process_enrollment_response(enrollment, response)
     unless response.is_a?(Hash)
       handle_response_is_not_a_hash(enrollment)
       return
     end
 
+    # We want to deactivate them regardless of status:
+    if fraud_pending?(enrollment)
+      handle_fraud_review_pending(enrollment)
+    end
+
     case response['status']
```

Comment:
Just checking my understanding here: if fraud is pending, we call `handle_fraud_review_pending` and then move on to the `when` statement? Which case then does a pending fraudy enrollment end up in?

Reply:
Yes. Can I make this clearer in the code?

The idea is that we process (1) their `fraud_pending?` status, and then (2) `response['status']`.

The two are _mostly_ unrelated, except that the successfully-proofed case needs to not activate them if they're `fraud_pending?`. But putting that logic in each of the `when` statements felt really repetitive (and error-prone if we ever added another one).

Reply:
I'm kinda just thinking about making it clear that if fraud is detected, that the enrollment in question then gets funneled into the correct pipeline.

Reply:
I updated the comment above to try to better capture this. Do you think this makes it clear, or can we do more?